src/components/PendingBusiness/PendingBusiness.jsx

```diff
@@ -0,0 +1,330 @@
+/* eslint-disable react/prop-types */
+import { useEffect, useState } from 'react';
+import { useBackend } from '../../contexts/BackendContext';
+import { useNavigate } from 'react-router-dom';
+import { renderEmail } from 'react-html-email';
+import approvedEmailTemplate from '../../templates/ApprovedBusinessTemplate';
+import deniedEmailTemplate from '../../templates/DeniedBusinessEmailTemplate';
+import {
+  Text,
+  Flex,
+  IconButton,
+  Box,
+  Breadcrumb,
+  BreadcrumbLink,
+  BreadcrumbItem,
+  Card,
+  CardHeader,
+  CardBody,
+  Heading,
+  Td,
+  Tr,
+  Button,
+  Table,
+  Thead,
+  TableContainer,
+  Select,
+  SimpleGrid,
+  Tbody,
+  ButtonGroup,
+  useToast,
+} from '@chakra-ui/react';
+const PendingBusiness = ({ data, handleHome }) => {
+  const { backend } = useBackend();
+  const navigate = useNavigate();
+  const toast = useToast();
+
+  const [residentialStatus, setResidentialStatus] = useState();
+
+  useEffect(() => {
+    const getResidentialStatus = async () => {
+      try {
+        const businessResponse = await backend.get(`/business/${data.id}`);
+        const residentialStatus = businessResponse.data[0].residential;
+        if (!residentialStatus) {
+          setResidentialStatus("PENDING");
+        } else {
+          setResidentialStatus(residentialStatus.toUpperCase());
+        }
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+    getResidentialStatus();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'backend' and 'data.id'. Either include them or remove the dependency array.